### ISSUE TITLE
Can't set games addons in shortcut

### ISSUE THREAD

**Issue by pasnox** [realcopacetic/skin.copacetic#114]
I do have a game addon and i can't set it easily in the home menu.
I had to cheat by adding it to favorites, then add a shortcut to this entry in the favorite, to then be kicked by kodi error:

```
2024-03-11 21:17:24.245 T:19076    info <general>: Skipped 5 duplicate messages..
2024-03-11 21:17:24.245 T:19076    info <general>: Loading skin file: MyGames.xml, load type: KEEP_IN_MEMORY
2024-03-11 21:17:24.245 T:19076   error <general>: Unable to load window XML: /storage/.kodi/addons/skin.copacetic/16x9/MyGames.xml. Line 0
                                                   Failed to open file
```

**Comment by realcopacetic** [comment]
Hi I currently don't have support for MyGames.xml. I will add this. Can you recommend a game addon I can use to test your use case please @pasnox ?

**Comment by pasnox** [comment]
Not really, I do use a custom one based on one for an old Kodi version. I can share it with you though, even if it may not work because you would not have the needed binaries.

**Comment by pasnox** [comment]
> Hi I currently don't have support for MyGames.xml. I will add this. Can you recommend a game addon I can use to test your use case please @pasnox ?

You can use the RetroPlayer addon and the snes backend for it. There are free opensource snes games you can grab online.

I cna also share the addon i'm using but it's a bit special, it's jsut an addon that basically kill kodi and starts RetroArch own build.
